packages/blockchain-wallet-v4-frontend/src/data/components/exchange/reducers.ts

```diff
@@ -0,0 +1,79 @@
+import * as AT from './actionTypes'
+import { ExchangeActionTypes, ExchangeState } from './types'
+import Remote from 'blockchain-wallet-v4/src/remote/remote'
+
+const INITIAL_STATE: ExchangeState = {
+  limits: Remote.NotAsked,
+  min: null,
+  max: null,
+  // @PHIL how to solve for default state for all these, 15-28
```

Comment:
@TheLeoB I don't think the INITIAL_STATE should change from https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2102/files#diff-80ce3fa62bee1d23850e68184db9e267L5-L13.

If the value can be null we should reflect that in `ExchangeState ` here https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2102/files#diff-3c02af347613331721c9487161a17413R37-R49

Reply:
Made the change to types file. targetFee was removed from initial state since it wasn't being used.